tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/SupplierSectionSteps.cs

```diff
@@ -104,15 +104,16 @@ public async Task ThenTheResponseContainsTheFollowingSupplierAddress(Table table
         public async Task WhenTheUserMakesARequestToUpdateTheSupplierWithOrderId(string orderId, Table table)
         {
             var supplierTable = table.CreateInstance<SupplierSectionTable>();
-            var addressTable = _context["SupplierAddress"];
-            var contactTable = _context["SupplierContact"];
+            
+            _context.TryGetValue("SupplierAddress", out var address);

```

Comment:
For consistency reasons want to ensure the code is using the same key e.g. "SupplierAddress" to retrieve a value as well as setting a value. Could `SupplierAddress` & `SupplierContact` be changed to use constants? It should be the same constant wherever the data is added too.

Reply:
👍